docs/source/references/schema.json

```diff
@@ -1070,6 +1063,16 @@
       ]
     }
   },
+  "iroha_data_model::block::committed::FinishedBlockHash": {
+    "Struct": {
+      "declarations": [
+        {
+          "name": "internal",
+          "ty": "iroha_crypto::hash::Hash"
+        }
+      ]
+    }
+  },
```

Comment:
Shouldn't this will be default hash for queries from SDK libraries? If yes, then this will complicate the usage from their side. Probably better to remove `internal` from public interface. I.e. using `PerfectHash(Hash)` or manually implementing `IntoSchema` and (de-)serialization

Reply:
You are correct and that is why I went with the partial hash scheme.